app/Http/Controllers/LdapController.php

```diff
@@ -5,37 +5,29 @@
 class LdapController extends Controller
 {
     /**
-     * @param  string  $query
-     * @param  bool  $only_active
      * @return array
      */
-    public static function searchUtwente($query, $only_active = false)
+    public static function searchUtwente(string $query, bool $only_active = false)
     {
         $response = file_get_contents(sprintf('%s?key=%s&filter=(%s)', config('ldap.proxy.utwente.url'), config('ldap.proxy.utwente.key'), urlencode($query)));
         $result = json_decode($response)->result;
 
         if ($only_active) {
-            return array_filter($result, function ($row) {
-                if ($row->active) {
-                    return true;
-                }
-
-                return false;
-            });
+            return array_filter($result, static fn ($row): bool => (bool) $row->active);
         }
 
-        return $result;
+        return $result ?? [];
```

Comment:
might break something down the line

Reply:
it gets checked at all instances of checkutwente accounts